rules/options.go

```diff
@@ -3,11 +3,14 @@ package rules
 import (
 	"time"
 
+	"github.com/IBM-Cloud/go-etcd-rules/internal/jitter"
 	"golang.org/x/net/context"
 )
 
 const (
 	defaultRuleID = "NO_ID_SET"
+
+	syncIntervalJitterPercent = 0.1
```

Comment:
Jitter is zero when the interval or delay is zero, since `0±X%` is always `0`. I added some tests here to make that more evident: https://github.com/IBM-Cloud/go-etcd-rules/pull/134/files#diff-646ed9a017cf346d5abdd1bfc857ccc9c6b02e76d2206f2775e4a795aa4cee44R23-R28
```go
		{
			description:      "zero value",
			duration:         DurationGenerator{},
			expectLowerBound: 0,
			expectUpperBound: 0,
		},
```